R/dictionaries.R

```diff
@@ -70,11 +70,11 @@ print_dictionary <- function(entry, level = 1) {
 
 
 # Internal function for special handling of multi-word dicitionary values
```

Comment:
Could you please document these and the other dictionary functions? OK to make this a separate PR.

Reply:
I will do separately